core/module.h

```diff
@@ -234,6 +233,28 @@ class Module {
     return attrs_;
   }
 
+  const std::vector<Task *> tasks() const {
+    return tasks_;
+  }
+
+  void set_attr_offset(size_t idx, bess::metadata::mt_offset_t offset) {
+    if (idx < bess::metadata::kMaxAttrsPerModule) {
+      attr_offsets_[idx] = offset;
+    }
+  }
+
+  const bess::metadata::mt_offset_t *all_attr_offsets() const {
```

Comment:
If it is used in the form of `all_attr_offsets()[i])` in most cases, how about using `mt_offset_t attr_offset(size_t idx)` instead?